python/ray/data/tests/test_stats.py

```diff
@@ -152,6 +152,12 @@ def enable_get_object_locations_flag():
     ctx.enable_get_object_locations_for_metrics = True
 
 
+@pytest.fixture(autouse=True)
```

Comment:
if we want verbose stats logs to be disabled by default, we should not enable `autouse`, since that has the effect of setting `ctx.verbose_stats_logs = True` for all unit tests. Instead, we could add as an arg to `test_dataset_hide_verbose_log` (and any other tests which need this configured) or just manually set the `DataContext` like you are doing currently.

(likely a bunch of test and docs outputs will need to be updated to remove `extra_metrics`)

Reply:
Yeah I did this mostly to minimize the testing impact, I can go through and update the tests though.